### PR TITLE
Add Japanese translation of "Getting started over HTTP" and "Getting started" documents

### PR DESCRIPTION
**Pull request by hirosassa**
(Describe the goal of this PR. Mention any related Issue numbers)

### Category (place an `x` in each of the `[ ]`)

* [ ] `slack_bolt.App` and/or its core components
* [ ] `slack_bolt.async_app.AsyncApp` and/or its core components
* [ ] Adapters in `slack_bolt.adapter`
* [x] Document pages under `/docs`
* [ ] Others

## Requirements (place an `x` in each `[ ]`)

Please read the [Contributing guidelines](https://github.com/slackapi/bolt-python/blob/main/.github/contributing.md) and [Code of Conduct](https://slackhq.github.io/code-of-conduct) before creating this issue or pull request. By submitting, you are agreeing to those rules.

* [x] I've read and understood the [Contributing Guidelines](https://github.com/slackapi/bolt-python/blob/main/.github/contributing.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).
* [ ] I've run `./scripts/install_all_and_run_tests.sh` after making the changes.

### CODE REVIEW

**Comment by srajiang** [comment]
Woot, big thank you @hirosassa for this contribution 🙌 ! 
I will let @seratch take the lead in providing review on this.

File: docs/_tutorials/ja_getting_started.md [hunk 1]
```diff
@@ -79,18 +88,21 @@ which python3
...
 ```
 
-2. **OAuth & Permissions ページのボットトークン（xoxb）をコピー**して、別の環境変数に保存します。
+2. **OAuth & Permissions ページのアプリレベルトークン（xapp）をコピー**して、別の環境変数に保存します。
 ```shell
-export SLACK_BOT_TOKEN=xoxb-<ボットトークン>
+export SLACK_APP_TOKEN=<your-app-level-token>
 ```
 
+> 🔒 全てのトークンは安全に保管してください。最低限、パブリックなバージョンコントロールにチェックインすることは避けてください。また、上記の例のように環境変数を介してアクセスするようにしてください。詳細な情報は [best practices for app security](https://api.slack.com/authentication/best-practices).のドキュメントを参照してください。
```
Comment (seratch) [inline docs/_tutorials/ja_getting_started.md:103]:
```suggestion
> 🔒 全てのトークンは安全に保管してください。少なくともパブリックなバージョン管理にチェックインするようなことは避けるべきでしょう。また、上にあった例のように環境変数を介してアクセスするようにしてください。詳細な情報は [アプリのセキュリティのベストプラクティス](https://api.slack.com/authentication/best-practices)のドキュメントを参照してください。
```

File: docs/_tutorials/ja_getting_started.md [hunk 2]
```diff
@@ -102,57 +114,42 @@ pip install slack_bolt
...
+app = App(token=os.environ.get("SLACK_BOT_TOKEN"))
 
 # アプリを起動します
 if __name__ == "__main__":
-    app.start(port=int(os.environ.get("PORT", 3000)))
+    SocketModeHandler(app, os.environ["SLACK_APP_TOKEN"]).start()
 ```
 
-このようにトークンと署名シークレットだけあれば、最初の Bolt アプリが作成できます。「`app.py`」ファイルを保存して、コマンドラインで以下を実行します。
+このようにトークンがあれば、最初の Bolt アプリが作成できます。「`app.py`」ファイルを保存して、コマンドラインで以下を実行します。
```
Comment (seratch) [inline docs/_tutorials/ja_getting_started.md:127]:
```suggestion
このようにトークンさえあれば、最初の Bolt アプリを作成することができます。「`app.py`」ファイルを保存して、コマンドラインで以下を実行します。
```

File: docs/_tutorials/ja_getting_started.md [hunk 3]
```diff
@@ -102,57 +114,42 @@ pip install slack_bolt
...
 
 ### イベントを設定する
-アプリはワークスペース内の他のメンバーと同じように振る舞い、メッセージを投稿したり絵文字リアクションを追加したりできます。Slack ワークスペースで発生するイベント（メッセージが投稿されたときや、メッセージに対するリアクションがつけられたときなど）をリッスンするには、[Events API を使って特定の種類のイベントをサブスクライブします](https://api.slack.com/events-api)。
+アプリはワークスペース内の他のメンバーと同じように振る舞い、メッセージを投稿したり、絵文字リアクションを追加したり、イベントをリッスンして返答したりできます。
 
-アプリをイベントに対応させるため、まずこのアプリの設定ページに戻って設定を行います。[アプリ管理ページ](https://api.slack.com/apps)でアプリをクリックします。次に、左サイドバーの「**Event Subscriptions**」をクリックします。「**Enable Events**」というラベルのスイッチをオンに切り替えます。
+Slack ワークスペースで発生するイベント（メッセージが投稿されたときや、メッセージに対するリアクションがつけられたときなど）をリッスンするには、[Events API を使って特定の種類のイベントをサブスクライブします](https://api.slack.com/events-api)。
 
-「**Request URL**」というラベルのテキスト入力フィールドが表示されます。Request URL は、指定したイベントに対応する Slack からの HTTP POST リクエストの送信先となるパブリック URL です。
+> 💡 このチュートリアルの序盤でソケットモードを有効にしました。ソケットモードを使うことで、アプリがパブリックなHTTPエンドポイントを公開せず、イベントAPIやインタラクティブコンポーネントを利用できるようになります。このことは、開発時やファイヤーウォールの裏からのリクエストを受ける際に便利です。HTTPはホスティングされた環境にデプロイされたアプリや、Slack App Directoryで配布されるアプリに向いています。これらの情報については、[このドキュメント](/bolt-python/tutorial/getting-started-http).を参照してください。
```
Comment (seratch) [inline docs/_tutorials/ja_getting_started.md:142]:
```suggestion
> 💡 このチュートリアルの序盤でソケットモードを有効にしました。ソケットモードを使うことで、アプリが公開された HTTP エンドポイントを公開せずに Events API やインタラクティブコンポーネントを利用できるようになります。このことは、開発時やファイヤーウォールの裏からのリクエストを受ける際に便利です。HTTP での方式はホスティング環境にデプロイするアプリや Slack App Directoryで配布されるアプリに適しています。HTTP での情報については[このドキュメント](/bolt-python/tutorial/getting-started-http)を参照してください。
```

File: docs/_tutorials/ja_getting_started.md [hunk 4]
```diff
@@ -102,57 +114,42 @@ pip install slack_bolt
...
 
-> ⚙️ [API サイト](https://api.slack.com/docs/hosting)に、Slack の開発チームがアプリのホストによく使用する一般的なホスティングプロバイダをいくつか記載しておきました。
+それでは、Slackにどのイベントをリッスンするかを教えてあげましょう。
```
Comment (seratch) [inline docs/_tutorials/ja_getting_started.md:144]:
```suggestion
それでは、私たちがどのイベントをリッスンしたいかを Slack に伝えましょう。
```

File: docs/_tutorials/ja_getting_started.md [hunk 5]
```diff
@@ -194,27 +189,25 @@ if __name__ == "__main__":
 
 ### アクションを送信して応答する
 
-インタラクティブ機能を有効にすると、ボタン、選択メニュー、日付ピッカー、モーダル、ショートカットなどの機能が利用できるようになります。イベントと同様に、Slack からのアクション（*ユーザーがボタンをクリックした*など）の送信先となる URL を設定する必要があります。
+インタラクティブ機能を有効にすると、ボタン、選択メニュー、日付ピッカー、モーダル、ショートカットなどの機能が利用できるようになります。アプリ設定ページの「**Interactivity & Shortcuts**」を見てみましょう。
```
Comment (seratch) [inline docs/_tutorials/ja_getting_started.md:192]:
```suggestion
インタラクティブ機能を有効にすると、ボタン、選択メニュー、日付ピッカー、モーダル、ショートカットなどの機能が利用できるようになります。アプリ設定ページの「**Interactivity & Shortcuts**」にアクセスしてください。
```

File: docs/_tutorials/ja_getting_started.md [hunk 6]
```diff
@@ -194,27 +189,25 @@ if __name__ == "__main__":
...
 
-アプリ設定ページに戻り、左サイドメニューの「**Interactivity & Shortcuts**」をクリックします。別の **Request URL** ボックスを見つけます。
+> 💡 ソケッットモードを有効にしていると、デフォルトで、基本的なインタラクティブ機能が有効になっていることに気づくでしょう。追加のアクションは不要です。HTTPを使っている場合は、Slackに対して、イベントを送信する為のリクエストURLを設定する必要があります。
```
Comment (seratch) [inline docs/_tutorials/ja_getting_started.md:194]:
```suggestion
> 💡 ソケッットモードを有効にしているとき、デフォルトで基本的なインタラクティブ機能が有効になっていることに気づくでしょう。追加のアクションは不要です。もし HTTP を使っている場合、Slack からのイベント送信先である Request URL を設定する必要があります。
```

File: docs/_tutorials/ja_getting_started.md [hunk 7]
```diff
@@ -194,27 +189,25 @@ if __name__ == "__main__":
...
 
-デフォルトでは、Bolt はイベントに使用しているのと同じエンドポイントをインタラクティブコンポーネントにも使用するように設定されているため、上記と同じリクエスト URL（この例では「`https://8e8ec2d7.ngrok.io/slack/events`」）を使用します。このままの状態で、右下隅にある「**Save Changes**」ボタンを押してください。これでインタラクティブ機能がアプリで利用できるようになりました。
+インタラクティブ機能が有効化されている時、ショートカット、モーダル、インタラクティブコンポーネント (ボタンや、選択メニュー、日付ピッカー) とのインタラクションはイベントとしてアプリに対して送信されます。
```
Comment (seratch) [inline docs/_tutorials/ja_getting_started.md:196]:
```suggestion
インタラクティビティが有効化されていれば、ショートカット、モーダル、インタラクティブコンポーネント (例：ボタン、選択メニュー、日付ピッカー) とのインタラクションはイベントとしてあなたのアプリに送信されます。
```

File: docs/_tutorials/ja_getting_started.md [hunk 8]
```diff
@@ -194,27 +189,25 @@ if __name__ == "__main__":
...
 
-![Request URL の設定](../../assets/request-url-config.png "Request URL の設定")
-
-それでは、アプリのコードに戻り、インタラクティブ機能を追加しましょう。インタラクティブ機能は 2 つのステップで構成されます。まず、ボタンを含んだメッセージをアプリから送信します。次に、ユーザーから返されるボタンクリックのアクションをリッスンし、それに応答します。
+それでは、アプリのコードに戻り、これらのイベントを処理する為のロジックを追加しましょう。
+- まず、ボタンを含んだメッセージをアプリから送信します。
```
Comment (seratch) [inline docs/_tutorials/ja_getting_started.md:199]:
```suggestion
- まず、インタラクティブコンポーネント（ここではボタン）を含んだメッセージをアプリから送信します。
```

File: docs/_tutorials/ja_getting_started.md [hunk 9]
```diff
@@ -235,9 +228,15 @@ def message_hello(message, say):
         text=f"Hey there <@{message['user']}>!"
     )
 
+@app.action("button_click")
+def action_button_click(body, ack, say):
+    # Acknowledge the action
```
Comment (seratch) [inline docs/_tutorials/ja_getting_started.md:233]:
```suggestion
    # アクションのリクエストを確認
```

**Review by seratch — CHANGES_REQUESTED** [review]
@hirosassa Thanks for submitting this awesome pull request! I hope you won't mind receiving these change requests 🙇

**Review by hirosassa — COMMENTED** [review]
@seratch Thanks for your review comments! Suggested changes are applied.

**Review by seratch — APPROVED** [review]
Looks great to me 👍

**Comment by hirosassa** [comment]
Thanks! 🚀

**Pull request merged**